File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Organization.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class Organization : DomainResource, IResource
{
    public Base.Address[]? Address { get; set; }
    public string? Name { get; set; }
    public Base.CodeableConcept[]? Type { get; set; }
    public string[]? Alias { get; set; }
    public bool? Active { get; set; }
    public Base.Identifier[]? Identifier { get; set; }
    public Base.ContactPoint[]? Telecom { get; set; }
    public Base.ResourceReference? PartOf { get; set; }
    public Base.ResourceReference[]? Endpoint { get; set; }
    public Organization_Contact[]? Contact { get; set; }

    public class Organization_Contact : BackboneElement
    {
        public Base.HumanName? Name { get; set; }
        public Base.Address? Address { get; set; }
        public Base.CodeableConcept? Purpose { get; set; }
        public Base.ContactPoint[]? Telecom { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Organization.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import Address
from base import CodeableConcept
from base import ContactPoint
from base import HumanName
from base import DomainResource
from base import Reference
from base import Identifier
from base import BackboneElement

@dataclass(kw_only=True)
class OrganizationContact(BackboneElement):
    name: Optional[HumanName] = None
    address: Optional[Address] = None
    purpose: Optional[CodeableConcept] = None
    telecom: Optional[List[ContactPoint]] = field(default_factory=list)

@dataclass(kw_only=True)
class Organization(DomainResource):
    address: Optional[List[Address]] = field(default_factory=list)
    name: Optional[str] = None
    type: Optional[List[CodeableConcept]] = field(default_factory=list)
    alias: Optional[List[str]] = field(default_factory=list)
    active: Optional[bool] = None
    identifier: Optional[List[Identifier]] = field(default_factory=list)
    telecom: Optional[List[ContactPoint]] = field(default_factory=list)
    part_of: Optional[Reference] = None
    endpoint: Optional[List[Reference]] = field(default_factory=list)
    contact: Optional[List[OrganizationContact]] = field(default_factory=list)